### ISSUE TITLE
CHAOSS Governance

### ISSUE THREAD

**Issue by yigakpoa** [chaoss/education#27]
Educate newbies on what CHAOSS’s Governance structure is. 

This includes information like the Board of Directors, Governance Document & any other information that we can share with the general public.

**Comment by geekygirldawn** [comment]
Happy to help! Is there a template for this, and how long should it be?

**Comment by yigakpoa** [comment]
Thank you @geekygirldawn. 

- @sgoggins will be creating a “background video template ” as referenced here: (https://github.com/chaoss/education/issues/23 ). But I think you can start and the video editors will fix that when it’s time…

- Do you mean the length of the video? If yes, I would say just go with the flow and the video editors will do their thing while editing the videos. Another option is that a module can have different videos. But i don't know what the larger team advises @ElizabethN @sgoggins @germonprez .

I hope this helps.

**Comment by peculiaruc** [comment]
> We might want to pause before recording any of the videos and put together a doc with guidelines for these videos so that we have some consistency. We did a project like this at VMware a while back and it was a bit of a disaster because we didn't have good guidelines and every video had a completely different look and feel along with overlapping content.
> 
> Do we want slides or no slides? If slides, we should have a slide template for people to use and the slides should be reviewed / approved before we record the video to make sure that everything looks nice.
> 
> What about demos? When should we have a demo instead of slides? What are some guidelines for screen size / font size?
> 
> Do we want people's faces in the videos or just voice?
> 
> Should we all use zoom to record? Should we have captions or no captions? Should we record with a transcript?
> 
> What else should we be thinking about?

Thank you @geekygirldawn , These are great points. I started putting this up as a guide.
https://docs.google.com/document/d/1EvtqQw6XQuW6grIl7dY_Prf3TUxBj_rt7an6dLDyVlo/edit. 
I am also working on the ReadMe and Contributing. md to sort these. I would like to have your input on the doc.
Do you have a sample of the slide deck you mentioned above, I would like to have a look.
Thank you.

**Comment by yigakpoa** [comment]
+1 to everything @geekygirldawn has said. 

Thank you for the doc @peculiaruc . I’m thinking when we’re done with the document, it will form the ReadMe and Contributing. md. 

Do you think we can finish this by tomorrow? So that contributors can start working on their videos as soon as possible. 

I will also extend the due dates on issues assigned to individuals to cover this new development.

**Comment by peculiaruc** [comment]
> +1 to everything @geekygirldawn has said. 
> 
> Thank you for the doc @peculiaruc . I’m thinking when we’re done with the document, it will form the ReadMe and Contributing. md. 
> 
> Do you think we can finish this by tomorrow? So that contributors can start working on their videos as soon as possible. 
> 
> I will also extend the due dates on issues assigned to individuals to cover this new development. 

The doc is for our collaboration. It needs input from some persons including you.
I had to wait for @geekygirldawn and @ElizabethN's input here too.

**Comment by peculiaruc** [comment]
> I'm going to guess that we might need to delay the start a little bit, since some of the people needed to review the guidelines / plan are in the US where it's a holiday week.
> 
> We can use the CHAOSS presentation templates: https://docs.google.com/presentation/d/1issLzBwMoVJybKFpB4hXR-DTdZ0SHVYADPkRSlhc1-I/edit#slide=id.g1385ab96aac_1_163 
> 
> I don't have the other examples I talked about because the VMware training was an internal training program.
> 
> Thanks to both of you for all of your work on this!
> 
> 

You are welcome @geekygirldawn

**Comment by yigakpoa** [comment]
Thank you @geekygirldawn & @peculiaruc 

Quick update:

I have continued working on the document and created a copy of the CHAOSS presentation template as advised in the main slide [.](https://docs.google.com/presentation/d/1eJDeGemnN39kZuALHHPIKUfEE8Kvnx6yhTestbfXo4c/edit) If it’s okay, we can use this for the Onboarding Course. I have also renamed the new copy to reflect its use for the Onboarding Course so we can avoid any confusions. 

I’m also thinking we can make the slide template flow similar to the Script (since the script will most likely feed the slide - If that makes sense). 

What do we all think? + @ElizabethN 

Thank you once again.
